services/121-service/test/metrics/export-list.test.ts

```diff
@@ -144,10 +136,31 @@ describe('Metric export list', () => {
     expect(data.length).toBe(1);
 
     const exportRegistration = data[0];
+    expect(exportRegistration.referenceId).toBe(
+      registrationScopedGoesPv.referenceId,
+    );
+  });
 
-    expect(exportRegistration).toMatchSnapshot({
-      created: expect.any(String),
-    });
+  it('should export all registration attributes when no "select" is provided', async () => {
+    // Act
+    const getRegistrationsResponse = await getServer()
+      .get(`/programs/${OcwProgramId}/metrics/export-list/registrations`)
+      .set('Cookie', [accessToken])
+      .send();
+
+    // Assert
+    const data = getRegistrationsResponse.body.data;
+    expect(getRegistrationsResponse.status).toBe(HttpStatus.OK);
+    expect(data.length).toBe(5);
+
+    for (const registration of registrationsOCW.slice(1)) {
```

Comment:
Is saw that this was also in the previous version of the tests so maybe out of scope for this PR. But do you know why this slice is needed?

Reply:
Nope - I just carried it over from the previous test. I didn't want to make the scope of this any bigger given that it seems to be urgent.